### DISCUSSION TITLE
Schedule for Posting the solutions to the website 2023

### DISCUSSION THREAD

**Discussion by bishabosha** [scalacenter/scala-advent-of-code#348] [Announcements]
Thank you to everyone who voted in the doodle polls ([Dec 1 - 20](https://doodle.com/meeting/participate/id/bkoJ3Pvb), [Dec 21 - 25](https://doodle.com/meeting/participate/id/azn2GPOa))

I have written a schedule of who will post and when, based on the availability you provided, please confirm that you see this. (👍)

## Responsibility

You should write a description of the solution, and provide the code in the main body of the article, as in these examples:
- [day 1 2023](https://scalacenter.github.io/scala-advent-of-code/2023/puzzles/day01)
- [day 2 2023](https://scalacenter.github.io/scala-advent-of-code/2023/puzzles/day02)

*Please make sure that you do both part 1 and part 2 of the challenge before you write the article*

## Schedule

| Day | Author | Published? |
|-----|--------|------------|
| 1 (Fri) | @sjrd  | ✅ |
| 2 (Sat) | @bishabosha   | ✅ |
| 3 (Sun) | @bishabosha, @iusildra  | ✅ |
| 4 (Mon) | @shardulc  | ✅ |
| 5 (Tue) | ~natsukagami~ ~stewSquared~ @GrigoriiBerezin | ✅ |
| 6 (Wed) | ~bishabosha~ @spamegg1 | ✅ |
| 7 (Thu) | @anatoliykmetyuk  | ✅ |
| 8 (Fri) | ~iusildra~ @prinsniels | ✅ |
| 9 (Sat) | @SethTisue  | ✅ |
| 10 (Sun) | @EugeneFlesselle | ✅ |
| 11 (Mon) | ~stewSquared~ @natsukagami | ✅ |
| 12 (Tue) | @mbovel  | ✅ |
| 13 (Wed) | @adpi2 | ✅ |
| 14 (Thu) | @anatoliykmetyuk  | ✅ |
| 15 (Fri) | @sjrd | ✅ |
| 16 (Sat) | @iusildra | ✅ |
| 17 (Sun) | @stewSquared | ✅ |
| 18 (Mon) | @EugeneFlesselle | ✅ |
| 19 (Tue) | @mbovel | ✅ |
| 20 (Wed) | ~adpi2~ @merlinorg | ✅ |
| 21 (Thu) | @alexandru | 💭 |
| 22 (Fri) | @natsukagami | 💭 |
| 23 (Sat) | @stewSquared | ✅ |
| 24 (Sun) | ~bishabosha~ @merlinorg  | ✅ |
| 25 (Mon) | @bishabosha | ✅ |

**Comment by bishabosha** [comment]
if anyone else sees this and suddenly wants to volunteer then feel free to request here.

**Reply by shardulc** [reply]
If needed, I can tag team with you for one or both of the last days. (I am not confident in my ability to solve them alone!)

**Reply by iusildra** [reply]
That would be fun to do some pair programming for the last days (even though I don't know if I'll be able to attend)

**Reply by merlinorg** [reply]
Happy to volunteer if a volunteer is needed for a day.

**Reply by bishabosha** [reply]
@merlinorg thanks for volunteering! maybe you could do 24th? or unless someone else wants to drop (e.g. @adpi2 for 20th)

**Reply by adpi2** [reply]
I'll be happy to drop day 20th if you want to do it @merlinorg

**Reply by merlinorg** [reply]
@bishabosha I'd be up for either/both.

**Reply by bishabosha** [reply]
Ok @merlinorg you can have both 🙌, thanks!

**Comment by bishabosha** [comment]
@stewSquared has volunteered to write the article for day 5 later today - I have swapped the dates around accordingly @natsukagami

**Comment by SethTisue** [comment]
@mbovel do you still intend to do day 12?

**Reply by mbovel** [reply]
Yes; I will finish it as son as possible. Sorry for the delay!

**Reply by SethTisue** [reply]
no worries, just checking :-)

**Comment by bishabosha** [comment]
What do people want to do in 2024? I am no longer with the Scala Center, does anyone want to volunteer to push this?

**Reply by adpi2** [reply]
I can do the maintenance work, but it would be great if someone volunteers to do the scheduling, and to solve the puzzles.

**Reply by mbovel** [reply]
I'd be happy to do 2 days!

**Reply by mbovel** [reply]
@samuelchassot can do 1

**Reply by shardulc** [reply]
@kacperfkorban do you want to do one?

**Reply by shardulc** [reply]
@Dylan-vrl ?

**Reply by KacperFKorban** [reply]
I can do one.

**Reply by EugeneFlesselle** [reply]
I can do one as well 🙌

**Reply by dyvrl** [reply]
I can do one 🙂

**Reply by KacperFKorban** [reply]
Hi, I created a separate post for this year: https://github.com/scalacenter/scala-advent-of-code/discussions/571
It would be great if you could post concrete days to which you want to be assigned.

**Reply by SethTisue** [reply]
I'll be doing the scheduling this year, starting soon.

**Reply by bishabosha** [reply]
I can do some Wednesdays

**Comment by shardulc** [comment]
I can solve a couple

**Comment by SethTisue** [comment]
I can do a day, also happy to do 2 days if we come up a bit short on volunteers. I usually bail around day 15 or 20 when the problems get too time-consuming

**Comment by SethTisue** [comment]
2024 scheduling is at https://github.com/scalacenter/scala-advent-of-code/discussions/571
